apps/passport-client/new-components/screens/Home/NewHomeScreen.tsx

```diff
@@ -284,37 +241,8 @@ export const NewHomeScreen = (): ReactElement => {
     }
   });
 
-  // variable event gap for mobile view and desktop view
-  const eventGap =
-    windowWidth > MAX_WIDTH_SCREEN ? EVENT_GAP_DESKTOP : EVENT_GAP;
-  console.log(eventCardWidth);
-  useLayoutEffect(() => {
-    console.log(scrollRef.current, pcdCardScrollRef.current);
-    if (scrollRef.current) {
-      setEventCardWidth(
-        calculateElWidth(
-          scrollRef.current.scrollWidth,
-          eventGap,
-          tickets.length
-        )
-      );
-    }
-    if (pcdCardScrollRef.current) {
-      setTicketCardWidth(
-        calculateElWidth(
-          pcdCardScrollRef.current.scrollWidth,
-          TICKETS_HORIZONTAL_GAP,
-          tickets.length
-        )
-      );
-    }
-  }, [
-    eventGap,
-    setEventCardWidth,
-    setTicketCardWidth,
-    tickets.length,
-    isLoadedPCDs
-  ]);
+  const cardWidth =
+    (windowWidth > MAX_WIDTH_SCREEN ? MAX_WIDTH_SCREEN : windowWidth) - 40;
```

Comment:
40 is like margin?

Reply:
nice forgot to move it to a const, 40 is 20 + 20 - padding of the screen